core/src/banking_stage/transaction_scheduler/scheduler_controller.rs

```diff
@@ -1094,7 +1095,7 @@ mod tests {
                     &Keypair::new(),
                     &Pubkey::new_unique(),
                     1,
-                    i * 10,
+                    i * 10_000,
```

Comment:
why this change?

Reply:
(this commit is cherry-pick of #3931, which is requisite of this PR to pass CI. I'll rebase it after #3931 is merged, before make it officially review ready)

Answer is https://github.com/anza-xyz/agave/pull/3931#discussion_r1887031932